### ISSUE TITLE
Prior set

### ISSUE THREAD

**Issue by mdanb** [snap-stanford/pretrain-gnns#29]
Just want to make sure I understand exactly how you're using the prior set during supervised pretraining and training. From what I understand in the paper, the prior set is used during supervised pretraining along with the train and validation sets (how is it used here? As validation or train?) but it is not used during supervised training with fine-grained labels? 

EDIT:
Also, related to this question, I checked the code, and I see a "easy" and "hard" set for testing. I see that these are the human organism, but what I don't understand is why one is easy and the other is hard, if they're just randomly split between the two. i.e I'm referring to this line:
```
test_dataset_broad, test_dataset_none, _ = random_split(test_dataset, seed = args.seed, frac_train=0.5, frac_valid=0.5, frac_test=0)
```
